dev/game.py

```diff
@@ -1,10 +1,15 @@
 class Sprite():
-    def __init__(self, name, width=1, height=1):
+    def __init__(self, name, width=1, height=1, typ = None):
```

Comment:
Is `typ` a typo, or on purpose?

Reply:
Was on purpose as you cannot use the keyword 'type'